Update AXL manuscript

mcreixell opened the pull request:
Updated all figures and added their corresponding captions. Two outstanding items:

1. The idea of figure 2 is to provide an overview of the phosphoproteomic data using currently available tools. That's why I added an enrichment analysis with KSEA of PC9 WT vs KO. I was thinking of adding a volcano plot with p-values comparing both cell lines as well. It still looks incomplete to me, any suggestions?

2.  In figure 4, for some reason when I overlay both blots `YAP_WTvsKO_blot.svg` overrides `YAP_DR_blot.svg` and part of the former occupies the subplot. However, when I comment out one of the two to display only one at a time, both work as they're supposed to. Do you have any idea of why this is happening?

mcreixell commented on msresist/tests/test_CoClustering.py:
```diff
@@ -50,4 +50,4 @@ def test_pickle(distm):
 
     assert np.all(np.isfinite(unpickled.scores_))
-    np.testing.assert_allclose(MSC.scores_, scores, rtol=1e-2, atol=0.1)
+    np.testing.assert_allclose(MSC.scores_, scores, rtol=0.1, atol=0.2)
```
@aarmey I'm not sure if this is too high? This test was always failing and it still does some times with these new tolerance settings.

aarmey replied:
This should be fine. It looks like it fails due to a very small number of peptide assignments changing.

aarmey commented:
Regarding #1, a whole figure is a lot of space to dedicate to something we don't see as a major point of the paper. I'd rather that we quickly get to analysis of meaningful results.

I don't see figure 4 in the build? Perhaps you could post images here of when it is working with one placement, and broken with both?

aarmey commented on makefile:
```diff
@@ -1,4 +1,4 @@
-flist = 1 2 3 S3 M2 M3 M4 M5 MS2 MS3 MS4 MS5 MS6 MS7
+flist = 1 2 3 S1 S2 S3 S4 S5 S3 M2 M3 M4 M5 MS2 MS3 MS4 MS5 MS6 MS7
```
Add 4 here.

aarmey commented on msresist/figures/figure1.py:
```diff
@@ -30,7 +30,7 @@
 def makeFigure():
     """Get a list of the axis objects and create a figure"""
     # Get list of axis objects
```
The PCA results are just shoved in at the end, when I think they are really useful to synthesize all the phenotypic measurements. I'd suggest: (1) it's just the PCA results in Fig. 1, with select examples after, and all the phenotypic measurements are in the supplement, or (2) there's a figure on measuring various phenotypes, and then the PCA analysis is a separate Fig. 2.

There probably needs to be a cartoon at the beginning introducing the AXL KO and reintroduction, and small schematics describing each of the phenotypic measurements.

aarmey commented on msresist/figures/figure2.py:
```diff
@@ -16,7 +16,7 @@
 def makeFigure():
     """Get a list of the axis objects and create a figure"""
     # Get list of axis objects
```
Seems like the KSEA is obviously not very informative?

aarmey commented on msresist/figures/figureS1.py:
```diff
@@ -0,0 +1,29 @@
+"""
+This creates Supplemental Figure 1: Cell Viability time course
```
Seems to repeat quite a few plots in Fig. 1?

aarmey commented on msresist/figures/figureS2.py:
```diff
@@ -0,0 +1,29 @@
+"""
+This creates Supplemental Figure 2: Cell M time course
+"""
+
```
Same as with S1. Also, many journals limit the number of supplementary figures now to either supplementary figures must correspond to one of the main figures, or there can be no greater number than main figures. Should each phenotype have its own supplementary figure? Are you going to reference these plots in the text?

aarmey commented on msresist/figures/figureS3.py:
```diff
@@ -1,77 +1,30 @@
 """
-This creates Supplemental Figure 3: Prediction performance across different plsr models using different clustering strategies
+This creates Supplemental Figure 3: Cell Migration time course
```
Same here.

aarmey commented on msresist/figures/figureS4.py:
```diff
@@ -0,0 +1,44 @@
+"""
+This creates Supplemental Figure 4: Island effect across radiis and time points 
```
Same here.

aarmey commented on msresist/figures/figureS5.py:
```diff
@@ -0,0 +1,76 @@
+"""
+This creates Supplemental Figure 3: Prediction performance across different plsr models using different clustering strategies
```
I think this information should be in the main figure and we don't need a supplementary one.

- The performance versus component number can go—readers don't need to see this information if you're properly picking the right number of components.
- All the different methods can be presented in one plot using a stripplot like this: 

![](https://seaborn.pydata.org/_images/seaborn-stripplot-6.png)

mcreixell commented:
![figure4](https://user-images.githubusercontent.com/46428799/113515878-68649080-9577-11eb-84bb-ffeff14331e0.png)
![figure4-1](https://user-images.githubusercontent.com/46428799/113515932-b5e0fd80-9577-11eb-8598-f67473ed0aa8.png)
![figure4-2](https://user-images.githubusercontent.com/46428799/113515976-0f492c80-9578-11eb-8301-b9f096ce75c0.png)

mcreixell commented:
Working on the rest of the edits now

aarmey commented:
Not sure what's going on with those placements. We can work on merging this ignoring that, then I'll look into it. There's probably something weird about how the placement code is handling scaling that will take a bit to diagnose.

mcreixell commented:
Same thing happens with figure 1. I think this problem with the placements occurs when trying to overlay two .svg files contiguously. I couldn't come up with a diagram that made sense to describe the phenotypes but I think that cell viability / death are self-explanatory and that these images perfectly show what migration and island effect are.   

![figure1](https://user-images.githubusercontent.com/46428799/113532647-2f0a4000-95cc-11eb-855a-8650de321d5f.png)
![figure1-1](https://user-images.githubusercontent.com/46428799/113532651-303b6d00-95cc-11eb-9b8e-cf3a26482ba5.png)

mcreixell commented:
@aarmey I will update the figure captions once you've reviewed these last round of edits. I split the supplementals of the phenotypes in two figures because each time lapse panel consists of 10 plots x 4 phenotypes = 40 plots + 4 bar plots equals 44 plots. I split them in 22 (viability&death) and 22 (migration&island).
